conf/views/businessNamePage.njk

```diff
@@ -0,0 +1,56 @@
+{% extends "includes/layout.njk" %}
+
+{% from "govuk/components/back-link/macro.njk"     import govukBackLink %}
+{% from "govuk/components/input/macro.njk"         import govukInput %}
+{% from "govuk/components/button/macro.njk"        import govukButton %}
+{% from "govuk/components/error-summary/macro.njk" import govukErrorSummary %}
+{% from "macros/title.njk"                         import title %}
+
+{% block pageTitle %}
+  {{ title(messages("businessNamePage.title"), form.errors) }}
+{% endblock %}
+
+{% block mainContent %}
+
+  {{ govukBackLink({
+    text: messages("site.back")
+  }) }}
+
+  <div class="govuk-main-wrapper">
+    <div class="govuk-grid-row">
+      <div class="govuk-grid-column-two-thirds">
+
+        {% if form.errors.length > 0 %}
+          {{ govukErrorSummary({
+            "titleText": messages("error.summary.title"),
+            "errorList": form.errors
+          }) }}
+        {% endif %}
+
+        <form action="{{ routes.controllers.BusinessNamePageController.onSubmit(mode).url }}" method="post">
+
+          {{ csrf() | safe }}
+
+          {{ govukInput({
+            label: {
+              text: messages('businessNamePage.title'),
+              isPageHeading: true,
+              classes: "govuk-label--xl"
+            },
```

Comment:
Hint text is missing for business name and text input width needs to be shorter. e.g.
hint: {
 text: messages('hint message')
},
classes: "govuk-!-width-three-quarters",
id: ...

Reply:
added the hint guidance
reduced width but as this only extends to 50 chars, it may need to go back on review